docs/code_examples/systemd.service

```diff
@@ -0,0 +1,10 @@
+[Unit]
+Description=Start Errbot chatbot
+After=network.service
+
+[Service]
+Type=forking
+Environment="LC_ALL=en_US.UTF-8"
+ExecStart=/path/to/errbot/virtualenv/bin/errbot --daemon --config /path/to/errbot/config.py
```

Comment:
It's better not to daemonize and run in the foreground instead, letting systemd handle all that stuff transparently. That means `--daemon` can be removed here and `Type=forking` is no longer needed.